### PR TITLE
Cover config-template-ids parameter in factory.make_org (multiple values)

### LINKED ISSUE #2099: Add CLI test for config-template-ids parameter in factory.make_org (multiple values scenario)
Add CLI test to cover creation of organization (```factory.make_org()```) 
with multiple config templates, using ```config-template-ids``` parameter.
Amount of config-template-ids should be a random number between 2 and 5.
```
--config-template-ids CONFIG_TEMPLATE_IDS   Provisioning template IDs
                                            Comma separated list
                                            of values.
```
Issue is related to #1716

### PR DESCRIPTION
**Pull request by oshtaier**
Add CLI test for config-template-ids parameter in factory.make_org (multiple values scenario)

Closes #2099

```
nosetests tests/foreman/cli/test_org.py -m test_add_multiple_configtemplate_by_id
.
----------------------------------------------------------------------
Ran 1 test in 56.271s

OK
```

### CODE REVIEW

**Comment by Ichimonji10** [comment]
ACK

File: tests/foreman/cli/test_org.py [hunk 1]
```diff
@@ -16,6 +16,8 @@
     data, run_only_on, skip_if_bug_open, stubbed)
 from robottelo.test import CLITestCase
 
+import random
```
Comment (elyezer) [inline tests/foreman/cli/test_org.py]: nitpick: move the `import` statement lines before `from ... import ...` lines and separate the blocks by one blank line.

**Comment by elyezer** [comment]
ACK, pending nitpick.

**Comment by elyezer** [comment]
ACK

**Pull request merged**